### PR TITLE
chore: add .github/replies.yml

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## PR Checklist

- [x] Addresses an existing open issue: starts on #5547
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/typescript-eslint/typescript-eslint/blob/main/CONTRIBUTING.md) were taken

## Overview

Adds a first auto-reply based on https://github.com/typescript-eslint/typescript-eslint/issues/5538#issuecomment-1226118930.

### CODE REVIEW

File: .github/replies.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+replies:
+  - body: |
+      <img alt="Maury meme: You checked 'I have searched for related issues'. A cursory issue search related issues determined that was a lie." src="https://user-images.githubusercontent.com/3335181/186501430-7f4c481a-f557-48b0-826c-74bb8434f4d5.png" height="483" width="488">\
+      Thanks for posting, but this is a duplicate of #<insert issue>. Next time, please use the search 🙂.
```
Comment (JoshuaKGoldberg) [inline .github/replies.yml:4]: Is this too passive aggressive for general use? 🤔

Comment (bradzacher) [reply .github/replies.yml:4]: Yeah whilst I love memes - I think that the meme is a bit too passive-aggressive.

**Comment by bradzacher** [comment]
My saved replies right now:

# Extension Rule

```
Thanks for the issue!

This is an extension rule. That means we take the base rule and extend it to support TS syntax or features.

We don't break from the base rule to add new features unless they are specifically related to TS.

In this instance - this request is not related to TS, thus we shall not implement it. If you'd like to see it, feel free to raise a request with the ESLint team.
```

# Fix has been merged

```
The fix has been merged to master, and will be released Monday, as per our [release schedule](https://github.com/typescript-eslint/typescript-eslint#versioning)

If you need it sooner, please try the `canary` tag on NPM.
```

# "Progress"

```
With any issue in opened in this project - it either has a visible progress in the form of an attached PR, or it has no progress.

We are a community run project. The **volunteer** maintainers spend most of their time triaging issues and reviewing PRs. This means that most issues *will not progress* unless a member of the community steps up and **champions** it.

If this issue is important to you - consider being that champion.

If not - please just subscribe to the issue and wait patiently.
Commenting asking for status updates does not bump issue priority in any way and just serves to spam ***everyone*** subscribed to the issue.
```

# "Progress - Nice"

```
We are a community run project. The **volunteer** maintainers spend most of their time triaging issues and reviewing PRs. This means that most issues *will not progress* unless a member of the community steps up and **champions** it.

If this issue is important to you - consider being that champion.
If not - please use the subscribe function and wait patiently for someone else to implement.
```

# Review Queue

```
This PR is in the queue of PRs to reviewed, and will be reviewed when we are able.

https://github.com/typescript-eslint/typescript-eslint/blob/master/CONTRIBUTING.md#addressing-feedback-and-beyond
```

# Use prettier

```
We ***STRONGLY*** recommend you use automatic code formatters like [prettier](https://www.npmjs.com/package/prettier).

Stylistic lint rules take a lot of effort to write, require ongoing maintenance to add more configuration as people want to fit their preferences, and are a large burden to maintain as new syntax is added.
The configurability also leads to huge fanout [making tests huge, and awful to maintain](https://github.com/typescript-eslint/typescript-eslint/blob/b814e635c1f34139c89e0176727480935dd45cac/packages/eslint-plugin/tests/rules/type-annotation-spacing.test.ts).

It's for these reasons that [eslint has frozen all stylistic lint rule options, and will not accept any new ones](https://eslint.org/blog/2020/05/changes-to-rules-policies).
```

File: .github/replies.yml [hunk 2]
```diff
@@ -3,3 +3,13 @@ replies:
...
       Thanks for posting, but this is a duplicate of #<insert issue>. Next time, please use the search 🙂.
     name: Clearly Duplicate Issue
+  - body: |
+      With any issue in opened in this project - it either has a visible progress in the form of an attached PR, or it has no progress.\
+      \
+      We are a community run project. The volunteer maintainers spend most of their time triaging issues and reviewing PRs. This means that most issues will not progress unless a member of the community steps up and champions it.\
+      \
+      If this issue is important to you - consider being that champion.\
+      \
+      If not - please just subscribe to the issue and wait patiently.\
```
Comment (Josh-Cena) [inline .github/replies.yml:30]:
```suggestion
      With any issue opened in this project — it either has visible progress in the form of an attached PR, or it has no progress.\
      \
      We are a community-run project. The volunteer maintainers spend most of their time triaging issues and reviewing PRs. This means that most issues will not progress unless a member of the community steps up and champions it.\
      \
      If this issue is important to you — consider being that champion.\
      \
      If not — please just subscribe to the issue and wait patiently.\
```

(My heart aches every time seeing an em-dash improperly typed)

File: .github/replies.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      Thanks for posting! This is a duplicate of #<insert issue>. Before filing an issue, please [use our issue search](https://github.com/typescript-eslint/typescript-eslint/issues) to check for open and closed issues that already address what you're looking for.
+    name: Clearly Duplicate Issue
+  - body: |
+      Thanks for the issue!
+      \
+      This is an extension rule. That means we take the base rule and extend it to support TS syntax or features.
+      \
+      We don't break from the base rule to add new features unless they are specifically related to TS.
+      \
+      In this instance - this request is not related to TS, thus we shall not implement it. If you'd like to see it, feel free to raise a request with the ESLint team.
```
Comment (Josh-Cena) [inline .github/replies.yml:12]:
```suggestion
      In this instance — this request is not related to TS, thus we shall not implement it. If you'd like to see it, feel free to raise a request with the ESLint team.
```

File: .github/replies.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      If this issue is important to you - consider being that champion.\
+      \
+      If not - please just subscribe to the issue and wait patiently.\
+      Commenting asking for status updates does not bump issue priority in any way and just serves to spam everyone subscribed to the issue.
+    name: Progress - Blunt
+  - body: |
+      We are a community run project. The **volunteer** maintainers spend most of their time triaging issues and reviewing PRs. This means that most issues *will not progress* unless a member of the community steps up and **champions** it.
+      \
+      If this issue is important to you - consider being that champion.
+      If not - please use the subscribe function and wait patiently for someone else to implement.
```
Comment (Josh-Cena) [inline .github/replies.yml:37]:
```suggestion
      If this issue is important to you — consider being that champion.
      If not — please use the subscribe function and wait patiently for someone else to implement this.
```

File: .github/replies.yml (hunk 2 above)
Comment (bradzacher) [reply .github/replies.yml]: > My heart aches every time seeing an em-dash improperly typed

add an em-dash key to my keyboard and I'll use it!!!

**Review by bradzacher — APPROVED** [review]

File: .github/replies.yml (hunk 2 above)
Comment (Josh-Cena) [reply .github/replies.yml]: On mac it's <kbd>Option</kbd><kbd>Shift</kbd><kbd>-</kbd>; I've heard it's notoriously hard to type em dashes on Windows...

**Pull request merged**